libs/blocks/icon-block/icon-block.css

```diff
@@ -103,14 +103,16 @@
   width: 60px;
 }
 
-.icon-block.inline.icon-height-32 .icon-area,
-.icon-block.inline.icon-height-32 .icon-area picture img {
-  height: 32px;
+.icon-block.inline.icon-height-40 .icon-area,
+.icon-block.inline.icon-height-40 .icon-area picture img {
+  min-width:40px;
+  height: 40px;
 }
 
-.icon-block.inline.icon-height-64 .icon-area,
-.icon-block.inline.icon-height-64 .icon-area picture img {
-  height: 64px;
+.icon-block.inline.icon-height-56 .icon-area,
+.icon-block.inline.icon-height-56 .icon-area picture img {
+  min-width: 56px;
+  height: 56px;
```

Comment:
Same with this. Any content that is using `icon-height-64` will need re-authored with this change.